services/core-web/src/components/common/CoreTable.tsx

```diff
@@ -9,6 +9,7 @@ interface CoreTableExpandConfig<T> extends ExpandableConfig<T> {
   rowKey?: string | ((record: any) => string);
   recordDescription?: string;
   subTableColumns?: ColumnType<any>[];
+  noSubTableExpandableRows?: boolean;
```

Comment:
Is there a reason not to just use rowExpandable = (record) => false? What is the goal here that this is serving?

Reply:
As discussed, this has been renamed to `matchChildColumnsToParent` and the purpose of this prop is because this PR's changes only applies to major project's final tab for the moment and having this prop is so we can keep using the current document table structure and slowly move to this PR's document table structure for other parts. So ultimately this prop is kind of temporary.